lib/suspenders/version.rb

```diff
@@ -1,9 +1,5 @@
 module Suspenders
-  RAILS_VERSION = "~> 7.0.0".freeze
-
-  minimum_ruby_version = "3.0.5"
-  maximum_ruby_version = Pathname(__dir__).join("../../.ruby-version").read.strip
-
-  RUBY_VERSION_RANGE = [minimum_ruby_version, maximum_ruby_version].freeze
-  VERSION = "20230113.0".freeze
+  VERSION = "3.0.0".freeze
+  RAILS_VERSION = "~> 7.0".freeze
```

Comment:
Since we're removing the executable, we don't need to be on the latest version of Rails. My thinking was, I don't want to prevent someone not on `>= 7.1` to be able to use the gem since it's just generators.

However, the only downside to that is if a generator uses a [feature](https://rubyonrails.org/2023/10/5/Rails-7-1-0-has-been-released) from `>= 7.1`, that feature won't work.